### ISSUE TITLE
I don't understand the issue with measurement

### ISSUE THREAD

**Issue by jimoleson** [ge0rg/aprsdroid#112]
Can you clarify why you do not allow imperial measurment? Can you explain? You sold this software in this region and  I think you need to support it. Would it necessitate  refactoring? What can I do to help?

**Comment by emiliogon** [comment]
I agree, all common units should be supported in regions the app is sold in.  The utility of this app is hampered for Imperial/US Standard users engaging in SAR, DF, geocaching, etc.

**Comment by ge0rg** [comment]
To be honest, this is a change affecting many aspects of the app (display on the Hub is only one part, the configuration of various elements is the other), and I just never had the time to implement it in a clean and useful way. I know that a large part of my user base is located in Imperial land, and it looks like I can't hope for a second attempt at converting the US of A to SI units. This is really not refusal to implement it, just a lack of time. The ticket response stating otherwise was created on April 1st, BTW.
